### PR TITLE
Adding placekey package

### PR DESCRIPTION
**Pull request by ecmascriptguru**
Hi @conda-forge/help-python

I hope you would like to review this pull request.

Thanks,
Ming

### CODE REVIEW

File: recipes/placekey/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 25235e7d5733ba229464fa4de0dacfeda84063b9fbb52041ac67d04c9c35d6be
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - backoff
```
Comment (mariusvniekerk) [inline recipes/placekey/meta.yaml:18]: are all of these deps required at build time?

File: recipes/placekey/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 25235e7d5733ba229464fa4de0dacfeda84063b9fbb52041ac67d04c9c35d6be
+
+build:
```
Comment (mariusvniekerk) [inline recipes/placekey/meta.yaml:12]: Can this package be `noarch: python`

File: recipes/placekey/meta.yaml (hunk 1 above)
Comment (ecmascriptguru) [reply recipes/placekey/meta.yaml:18]: Hi @mariusvniekerk 
Honestly I am not sure what should be required at build time. I just mentioned dependencies here.
Would you kindly let me know how to check if a specific package is required at build time?

Thanks,
Ming

Comment (mariusvniekerk) [reply recipes/placekey/meta.yaml:18]: Odds are you just need python and pip, i'd recommend removing all of them.  If you use a tool like [greyskull](https://github.com/conda-incubator/grayskull) to generate the recipe that will generally do this for you.

Comment (ecmascriptguru) [reply recipes/placekey/meta.yaml:18]: Okay, thanks. Let me try with greyskull.

**Comment by ecmascriptguru** [comment]
Hi @mariusvniekerk Would you like to check my updates again?
I used to the greyskull.

**Comment by ecmascriptguru** [comment]
Hi @mariusvniekerk 
Would you like to let me know when the package will be available at the conda-forge channel?

**Pull request merged**